modules/ibi-installer-installation-config.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/ibi-image-based-install.adoc 
+
+:_mod-docs-content-type: REFERENCE
+[id="ibi-installer-installation-config_{context}"]
+= ImageBasedInstallationConfig resource API specifications
```

Comment:
Same as [here](https://github.com/openshift/openshift-docs/pull/79610/files#r1754034359). I think we should convey that this is the specification of the `image-based-installation-config.yaml` configuration file.